test-files/ui/Makefile

```diff
@@ -46,11 +46,13 @@ endif
 
 clean:
 	#Remove test datasets from db
-	PGPASSWORD=$(PGPASSWORD) psql $(AUTH) -t -A -c "SELECT display_name FROM maps WHERE display_name like '%Cucumber%'" \
-	| while read mapname; do \
- 	   echo hoot delete-map $(HOOT_OPTS) $(DB_URL)/$$mapname | bash ; \
-	done; \
-	true
+	@if [[ `PGPASSWORD=$(PGPASSWORD) psql $(AUTH) -t -A -c "SELECT EXISTS (SELECT 1 FROM information_schema.tables WHERE table_name='maps');"` = 't' ]]; then \
+		PGPASSWORD=$(PGPASSWORD) psql $(AUTH) -t -A -c "SELECT display_name FROM maps WHERE display_name like '%Cucumber%'" \
+		| while read mapname; do \
+			echo hoot delete-map $(HOOT_OPTS) $(DB_URL)/$$mapname | bash ; \
+		done; \
+		true; \
+	fi
```

Comment:
This is a quick query to check if the `maps` table exists in the `information_schema.tables` table before querying for table names.